projects/core/src/occ/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './occ.module';
 export * from './services/index';
 export * from './utils/interceptor-util';
 export * from './adapters/index';
+export { USERID_ANONYMOUS as ANONYMOUS_USERID } from './utils/occ-constants';
```

Comment:
Can we decorate with a deprecation?

Reply:
Done. Exported `OCC_USERID_CURRENT` and `OCC_USERID_ANONYMOUS` in public API.